frontend/src/pages/RoomPage.tsx

```diff
@@ -257,27 +300,110 @@ export const RoomPage = () => {
         />
       </Stack>
       <Divider />
-      <Stack
-        direction="row"
-        spacing={2}
-        sx={{ width: "100%", flex: 1, minHeight: 0, p: 3 }}
-      >
-        <Stack spacing={2} sx={{ minWidth: "40%", maxWidth: "40%" }}>
-          <Box sx={{ flex: 1, minHeight: 0 }}>
-            <Question />
-          </Box>
-          <ChatProvider roomId={roomId || ""}>
-            <Box sx={{ height: "40%" }}>
-              <Chat />
+      <TabContext value={formType}>
```

Comment:
Should the `TabContext` only wrap the Question and not the chat or the editor? It would be weird to have such a large amount of space for the submissions while preventing users from viewing the editor or the chat.

Also, having the editor and the chat within the tab list makes them have to reconnect every time the user switches tabs.

Reply:
Logged as bug in #208